courses/free-courses-en.md

```diff
@@ -593,8 +593,9 @@
 
 #### Jenkins
 
-* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) -  CS Career Kaizen (Udemy)
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - CS Career Kaizen (Udemy)
 * [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Automation Step by Step
+* [Jenkins Devops](https://youtube.com/playlist?list=PL87padGs1bp9X95IxGQtFW44qWzywhbHM&si=tBgw4dwt8FYwobse) - Java Express
```

Comment:
Swap this with line 599 so that it is in alphabetical order, then you will pass the last check.